lib/beaker-hostgenerator/data/vmpooler.rb

```diff
@@ -35,6 +35,14 @@ module Vmpooler
           'platform' => 'el-7-x86_64',
           'template' => 'centos-7-x86_64'
         },
+        'cisco5-64' => {
+          'platform' => 'cisconxos-9-x86_64',
```

Comment:
So what beaker does with this platform string is splits it into three components:
* platform
* version
* arch

The regex used to do this is here:
https://github.com/puppetlabs/beaker/blob/54b1801d9bb6187796142160d3c0828639c542d5/lib/beaker/platform.rb#L6

So it looks like Beaker wouldn't actually recognize this platform string. Also, the version string would then be misleading since this is actually `cisco 5`, not `cisco 7`.

My recommendation here would be to set the platform string to `cisco-5-x86_64`.